code/modules/mining/equipment/lazarus_injector.dm

```diff
@@ -96,7 +96,7 @@
 	else
 		if("neutral" in S.faction)
 			S.forceMove(src)
-			S.name = "[M.name]'s [initial(S.name)]"
+			S.name = "[M.name]&#39s [initial(S.name)]"
```

Comment:
Just tested on live.

I can interact with a lazarus capsuled runtime without any issue. This make the name not display properly visually but fine in text panel.

On live, both were displaying just fine

Reply:
By interact I mean give karma or examine. If you try to give karma to your capsuled pet it will try to give the karma to you. Similarly, if you examine your capsuled pet it will examine you instead. Are you sure that's fine, because I replicated the issue myself when I was trying to fix it.

Reply:
This is so atypical, there is literally nothing else in the game that uses `&#39s` written out like this. I don't think this is a proper way to fix anything, I'm pretty sure it's related to #12338 which should be fixed in its own.